site/plugin-development.xml

```diff
@@ -196,21 +161,63 @@ $ ../scripts/rabbitmq-plugins enable rabbitmq_shovel</pre>
+              as well as <tt>rabbitmq-plugins.mk</tt> using erlang.mk
+              plugins facility. See below for a description of those
+              files.
+            </td>
+          </tr>
+          <tr>
+            <td><a href="&metronome_repo;/erlang.mk"><code>erlang.mk</code></a></td>
+            <td>
+              A local copy of <tt>erlang.mk</tt>. This is not a vanilla
+              copy because RabbitMQ relies on a few modifications
+              which have not been merged upstream at the time of
+              this writing. That's why <tt>ERLANG_MK_REPO</tt> and
+              <tt>ERLANG_MK_COMMIT</tt> are overridden for now.
             </td>
           </tr>
           <tr>
-            <td><a href="&metronome_repo;/package.mk"><code>package.mk</code></a></td>
+            <td><a href="&metronome_repo;/build.config"><code>build.config</code></a></td>
             <td>
-              Contains various settings used by the umbrella Makefile.
+              This file configures which features of erlang.mk should be
+              enabled. In most of RabbitMQ's plugins, EUnit is disabled,
+              because they depend on a running broker. Our erlang.mk
+              extensions take care of starting a node and run the EUnit
+              tests when you type <tt>make tests</tt>.
+            </td>
+          </tr>
+          <tr>
+            <td><a href="&metronome_repo;/rabbitmq-components.mk"><code>rabbitmq-components.mk</code></a></td>
+            <td>
+              A local copy of <tt>rabbitmq-components.mk</tt>. The
+              original file is in <tt>rabbitmq-common</tt> which your
+              plugin will depend on automatically. It contains other
+              erlang.mk extensions and helpers which must be defined
+              before <tt>erlang.mk</tt> inclusion. This file must be
+              kept up-to-date w.r.t. <tt>rabbitmq-common</tt>: when it
+              is out-of-date, you will get the following error:
+              <pre>error: rabbitmq-components.mk must be updated!</pre>
+              In this case, just run the following command to update
+              your copy:
+              <pre>make rabbitmq-components-mk</pre>
             </td>
           </tr>
           <tr>
             <td><a href="&metronome_repo;/src/rabbitmq_metronome.app.src"><code>src/rabbitmq_metronome.app.src</code></a></td>
             <td>
-              This file declares the dependencies required by the plugin (<code>rabbit</code>
-              and <code>amqp_client</code>). These will be available and started before starting the plugin. Note that the real app file is generated from this; there is no need to fill in the list of modules.
+              This file declares the dependencies required by the plugin
+              (<code>rabbit</code> and <code>amqp_client</code>). These
+              will be available and started before starting the plugin.
```

Comment:
Yes, "These" are dependencies/applications. I'm updating the patch to explicitely say "These applications".